Documentation/compatibility/wpf-accessibility-improvements.MD

```diff
@@ -0,0 +1,101 @@
+## 195: Accessibility improvements in WPF
+
+### Scope
+Major
+
+### Version Introduced
+4.7.1
+
+### Source Analyzer Status
+NotPlanned
+
+### Change Description
+__High Contrast improvements__
+
+- Focus for expander was not visible.  This is now fixed.
```

Comment:
If expander is the expander control, it should be:
expander --> an `T:System.Windows.Controls.Expander` control